File: listas de exercicio/lista 2/lista 2 casa C/questao 4.c
#include <stdio.h>

int main() {
    int a, b, c;
    int maior, menor, meio;

    printf("Digite o primeiro número: ");
    scanf("%d", &a);
    printf("Digite o segundo número: ");
    scanf("%d", &b);
    printf("Digite o terceiro número: ");
    scanf("%d", &c);

    if (a > b && a > c) {
        maior = a;
    } else if (b > a && b > c) {
        maior = b;
    } else {
        maior = c;
    }

    if (a < b && a < c) {
        menor = a;
    } else if (b < a && b < c) {
        menor = b;
    } else {
        menor = c;
    }

    if ((a > b && a < c) || (a < b && a > c)) {
        meio = a;
    } else if ((b > a && b < c) || (b < a && b > c)) {
        meio = b;
    } else {
        meio = c;
    }

    printf("Maior número: %d\n", maior);
    printf("Menor número: %d\n", menor);
    printf("Número do meio: %d\n", meio);
}

File: listas de exercicio/listas 2/lista 2 casa python/questao 4.py
A = input("Digite um numero:")
B = input("Digite um segundo numero:")
C = input("Digite um terceiro numero:")
A = float(A)
B = float(B)
C = float(C)
if A>B>C:
    print(f"o maior numero é {A} o menor numero é {C} e o numero do meio é {B}")
elif B>A>C:
    print(f"o maior numero é {B} o menor numero é {C} e o numero do meio é {A}")
elif C>A>B:
    print(f"o maior numero é {C} o menor numero é {B} e o numero do meio é {A}")
elif C>B>A:
    print(f"o maior numero é {C} o menor numero é {A} e o numero do meio é {B}")
elif B>C>A:
    print(f"o maior numero é {B} o menor numero é {A} e o numero do meio é {C}")
elif A>C>B:
    print(f"o maior numero é {A} o menor numero é {B} e o numero do meio é {C}")